draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -334,7 +334,9 @@ multiple values.
 ### Presentation format
 
 The presentation format for SvcFieldValue is a whitespace-separated
-list of the key=value pairs.  Each pair is presented in the following form:
+list of elements representing a key-value pair, with an absent value
+or "=" indicating an empty value.  Each element is presented in the
```

Comment:
I added notes to the keys that don't support empty values.  Hopefully that should be enough to make it clear.

I made this adjustment because `transport=quic no-default-transport` seemed a lot clearer than `transport=quic no-default-transport=""`.  In general, I think its worth investing in clarity for zone file editors, since zone files get edited a lot more often than DNS software.